### ISSUE TITLE
internal transmission

### ISSUE THREAD

**Issue by drhouse82** [mjhoptics/opticalglass#1]
Hi Michael, thanks for this very useful library!
I wondered if there is also data on internal transmission of the glasses and, exploring the API, I couldn't find any.
Is this correct?
Best
Joachim

**Comment by mjhoptics** [comment]
Hello Joachim,
You are correct, the API doesn’t include transmittance values. I believe it would be straightforward to do this as all of the current manufacturer spreadsheets have that information; sometimes for 2 different thicknesses.
The spreadsheets I’m using are in the data directory of the distribution. You can look in them and find out what the column header is for the transmittance data. You could use a similar approach to get the transmittance data as is used for the GlassCatalog glass_coefs method to return the transmittance data for the wavelengths contained in the catalog spreadsheet.
Are you interested in the transmittance data for a specific thickness or an absorption value that could be used for an arbitrary thickness? Would a function similar to rindex(), but for transmittance or absorption, be what you're looking for?
Thanks you for your interest and your kind comments.
Mike

**Comment by drhouse82** [comment]
Hi,
thanks for your reply. Yeah, a function like rindex would be perfect. I screened the code a bit, and in principle it should be straight-forward. However, as the different suppliers give transmission at different wavelengths, it is a bit less straight forward than with the coefficients. Lacking time, I have now solved my problem, by manually filtering the glasses with a list of glasses that have high transmission. With my next problem needing internal transmittances, I will likely implement the lookup and inform you.
Best
Joachim
